plugins/assimp/module/vtkF3DAssimpImporter.cxx

```diff
@@ -895,6 +895,7 @@ void vtkF3DAssimpImporter::UpdateTimeStep(double timestep)
 
   // get the animation tick
   timestep *= this->Internals->Scene->mAnimations[0]->mTicksPerSecond;
+  // TODO figure out how to make this work with a timevalue
```

Comment:
@Meakk this code worked with a timestep for some reasons. I need to figure out how to make it work with a time value in the time range. I'm sure it is simple. Feel free to take care of it and push on this branch.